packages/components/src/popover/README.md

```diff
@@ -227,4 +229,15 @@ Adjusts the size of the popover to prevent its contents from going out of view w
 
 _Note: this prop is deprecated and will be removed in WordPress 6.3. It has no effect on the component._
 
+-   Required: No
+
+### `variant`: `'alternate' | 'unstyled'`
+
+Specifies the popover's style.
+
+Leave undefined for the default style. Possible values are:
+-   `unstyled`:  The popover is essentially without any visible style, it has no background, border, outline or drop shadow, but the popover contents are still displayed.
+-   `alternate`: A style that has no elevation, but a high contrast with other elements.
+
+
```

Comment:
```suggestion
-   `alternate`: A style that has no elevation, but a high contrast with other elements.

```

Reply:
🤔  Made that change, but it didn't seem to fix the issue.

Reply:
Maybe I formatted the suggestion incorrectly — I was just trying to remove the double emtpy line 😅

Reply:
Ah, I see. When rendered in markdown it's becoming one list with the `-   Required: No` entry as the last item. I thought that's what you were referring to. No matter how many blank lines are there it still happens.

On Stack Overflow it was mentioned that you can use an HTML comment to separate lists, so I've done that. It's not a great solution. Visually it still looks a bit like one list.